### ISSUE TITLE
Blank/Frozen Video Recording on Chrome v87

### ISSUE THREAD

**Issue by kamalbennani** [cypress-io/cypress#9265]
### Current behavior
I think that the issue has been reported a couple of times but the context was quite different. 

So the issue here is that the Cypress Video Recording is Frozen or only shows the Cypress Frame Layout on Chrome V87. 

It's pretty straightforward to reproduce the issue, just download the latest version of Chrome (v87), write a small Cypress test, and run cypress using headless mode targeting chrome, and voilà.
 
For now, we've managed to partially resolve the issue by downgrading the Chrome version to v85.

![Screenshot 2020-11-20 at 22 25 36](https://user-images.githubusercontent.com/14129459/99851532-9379b500-2b7f-11eb-846d-2ad4c328ab9d.png)

### Versions

Cypress version: v5.6.0 and v5.2.0
Chrome version: V87

**Comment by randing89** [comment]
Looks like the same issue as I am getting here: https://github.com/cypress-io/cypress/issues/9267

**Comment by you1anna** [comment]
Also getting this issue, seems to be Chrome 87

**Comment by jweingarten** [comment]
I have the same thing. Cypress 5.6 and Chrome 87. I went back to Chrome 86 and it works again. 
On Chrome 87 you see this 
```
Finished processing: /foobar/tests    (0 seconds)
15:01:55                           /artifacts/videos/next.js.mp4   
```
It's always `0 seconds`

**Comment by mduft** [comment]
This started happening for me after upgrade to Cypress 6.0.0 which has Electron 86. Videos are no longer usable.

**Comment by forehalo** [comment]
Just found the reason:
TLDR, chromium throttles frame rate with `Page.screencastFrameAck`.

CDP.on('Page.screencastFrame', frame => {
  Page.screencastFrameAck({sessionId: frame.sessionId});
});
CDP.send('Page.startScreencast', { });

https://chromium.googlesource.com/chromium/src/+/1c762fa1266fda75231b9ac7b515318fc5328b4d
https://chromium.googlesource.com/chromium/src/+/1c762fa1266fda75231b9ac7b515318fc5328b4d/content/browser/devtools/protocol/page_handler.cc#881
https://chromium.googlesource.com/chromium/src/+/1c762fa1266fda75231b9ac7b515318fc5328b4d/content/browser/devtools/protocol/page_handler.cc#960

**Comment by jennifer-shehane** [comment]
I can recreate this issue. The video is blank/frozen when run in Chrome 87, which will now affect 6.0 users recording in Electron since that has been updated to Chrome 87.

- Chrome 87.0.4280.67 video: ❗️ **Blank** [spec.js.mp4.zip](https://github.com/cypress-io/cypress/files/5594339/spec.js.mp4.zip)
- Electron 87 video: ❗️ **Frozen** [spec.js.mp4.zip](https://github.com/cypress-io/cypress/files/5594345/spec.js.mp4.zip)
- Chromium 86.0.4240.80 video: ✅ **Working** 
[spec.js.mp4.zip](https://github.com/cypress-io/cypress/files/5594363/spec.js.mp4.zip)

**Comment by karanshah-browserstack** [comment]
The issue is also reproducible on edge 87 browser which was released recently.
